include/zephyr/sd/sd_spec.h

```diff
@@ -397,7 +417,7 @@ enum sdhc_clock_speed {
 	MMC_CLOCK_52MHZ = 52000000U,
 	MMC_CLOCK_DDR52 = 52000000U,
 	MMC_CLOCK_HS200 = 200000000U,
-	MMC_CLOCK_HS400 = 400000000U,
+	MMC_CLOCK_HS400 = 200000000U,
```

Comment:
Is this correct, seems like the MMC_CLOCK_HS200 and MMC_CLOCK_HS400 have the same value

Reply:
yes, same clock rate, 400 is ddr mode

Reply:
Might want to add a comment.